### PR TITLE
Add `protocol` to observability tags

### PR DESCRIPTION
**Pull request by NipunaMadhushan**
## Purpose
As a requirement for general observability features of Ballerina, we need to provide protocol for each service.

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/43518

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: native/src/main/java/io/ballerina/stdlib/websocket/observability/WebSocketObserverContext.java [hunk 1]
```diff
@@ -32,6 +32,7 @@ public class WebSocketObserverContext extends ObserverContext {
 
     private String connectionId = WebSocketObservabilityConstants.UNKNOWN;
     private String servicePathOrClientUrl = WebSocketObservabilityConstants.UNKNOWN;
+    private static final String protocol = "ws";
```
Comment (MohamedSabthar) [inline native/src/main/java/io/ballerina/stdlib/websocket/observability/WebSocketObserverContext.java:35]:
```suggestion
    private static final String PROTOCOL = "ws";
```

Comment (ayeshLK) [inline native/src/main/java/io/ballerina/stdlib/websocket/observability/WebSocketObserverContext.java:35]: Lets move the static variable to the top as it is the common practice.

Comment (NipunaMadhushan) [reply native/src/main/java/io/ballerina/stdlib/websocket/observability/WebSocketObserverContext.java]: @ayeshLK address review suggestions

**Review by chathurace — APPROVED** [review]

**Review by ayeshLK — APPROVED** [review]
LGTM

**Pull request merged**